### PR TITLE
Soviet's Playground

### PR DESCRIPTION
**Pull request by NeerOC**

### CODE REVIEW

File: Core/ObjectManager.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ class ObjectManager {
     // Invalidate 'me' if removed
     if (me && (!currentManager.localGuid || !newObjects.has(currentManager.localGuid.hash))) {
       me = null;
+    } else if (me) {
```
Comment (ejt1) [inline Core/ObjectManager.js]: remove debugging code

File: Extensions/CGActivePlayer.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Object.defineProperties(wow.CGActivePlayer.prototype, {
+
+  currentParty: {
+    /**
+     * Get the current active party if it exists, or return `undefined` if not in a party.
+     * @returns {wow.Party | undefined} - Returns the current party instance or undefined.
+     */
+    get: function () {
+      return (wow.Party?.currentParty) ? wow.Party.currentParty : undefined;
```
Comment (ejt1) [inline Extensions/CGActivePlayer.js]: `wow.Party` is a builtin class and will always exist, `wow.Party.currentParty` already returns undefined if player is not in a party.

Hence `wow.Party.currentParty` is enough

Comment (SovietFarmer) [reply Extensions/CGActivePlayer.js]: Done.

File: Core/ObjectManager.js (hunk 1 above)
Comment (SovietFarmer) [reply Core/ObjectManager.js]: Done.

**Review by ejt1 — APPROVED** [review]

**Pull request merged**